src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -88,9 +88,9 @@ public static MauiApp CreateMauiApp()
 					});
 
 #if NET6_0_OR_GREATER
-			appBuilder
-				.RegisterBlazorMauiWebView();
-			services.AddBlazorWebView();
+			services.AddMauiBlazorWebView();
+#if DEBUG
+			services.AddBlazorWebViewDeveloperTools();
```

Comment:
Related to this PR, should we consider a more generic name than `DeveloperTools`. Ex. `UseBlazorWebViewDevelopmentMode`. The scenario I had in mind was related to https://github.com/dotnet/maui/issues/4441 and how we were going to need some way to set `DETAILEDERRORS` in order to show exceptions within the UI (yellow bar) during development.

Reply:
DetailedErrors is orthogonal to the developer tools.

Reply:
It's orthogonal to the dev tools as a feature, however from the developer perspective, wouldn't it fall in the same bucket (ex. "stuff I need to make my development life easier"). It's also something that would be guarded by `#if DEBUG` just like this.

Reply:
Not really, you might want detailed errors in `#release` or based on some other setting that is independent of `#debug`